docs/global/modules/core/pages/reference/component-bulk-model-sync-gradle.adoc

```diff
@@ -132,10 +132,12 @@ mpsBuild {
 |`repositoryId`
 |String
 |Id of the target/source model-server repository.
+If the repository does not exist on the model-server, it will be created including a master branch.
 
 |`branchName`
 |String
 |Name of the target/source model-server branch.
+If the target branch does not exist on the model-server, it will be created.
```

Comment:
Can these changes be formed into individual feat commits? That way, they become visible in the changelog. This is important information for us as users as we can drop some workarounds now.

Reply:
It's hard to justify attributing these improvements as features to the gradle plugin, since they are a result of one bugfix in the model-client and no code in the plugin was changed for this. This would be one of those cases, where it would be nice to have the possibility to create some (additional) manual release notes.

Reply:
Ah I see. I thought this was a change in this PR.